src/docs/patterns/navigation.mdx

```diff
@@ -1,70 +1,220 @@
+
+<DemoCard
+    repository={'Drawer'}
+    angular={'https://pxblue-components.github.io/angular/?path=/story/components-drawer--within-a-drawer-layout'}
+    react={'https://pxblue-components.github.io/react/?path=/story/components-drawer--in-drawer-layout'}
 />
 
-## Fixed Navigation
+## Permanent
 
-The most basic navigation panel is fixed in place and always visible on the left side of the application. Note: a fixed navigation panel is not suitable when going to smaller resolutions or mobile devices.
+A permanent drawer is always visible and always open. The permanent drawer is typically full-width, as opposed to the rail variant (see below). Use this variation if you have:
 
-## Collapsible Navigation
+-   five or more top-level pages
+-   a nested navigation hierarchy
+-   enough space to justify it (typically desktop-only applications)
 
-A more common implementation of the navigation panel is a collapsible panel that can be opened and closed via the menu icon in the top left corner of the application. This implementation can have two different variations: content-resize or content-cover.
+<ImageGrid images={[NavigationPermanentImg]} regularWidth />
 
-### Content-Resize / Content-Push
+## Rail
 
-<ImageGrid images={[PermanentImg]} />
+A navigation rail is a permanently collapsed drawer. The rail is able to show less information than a standard drawer, but has the advantage of a more compact, space-saving package. Use this variation if you have:
 
-The content-resize implementation does not cover any of the main application content. Rather, the navigation panel slides in and the main application container resizes to accommodate (the navigation panel and main panel are side by side instead of one on top of the other).
+-   fewer than seven top-level pages
+-   a flat navigation hierarchy
+-   limited space - recommended for HMI devices
 
-### Content-Cover
+<ImageGrid images={[NavigationRailImg]} regularWidth />
 
-<ImageGrid images={[TemporaryImg]} />
+## Persistent
 
-<!-- @TODO switch side-navigation links over to the "Component Recommendations style" instead of a "Interactive Code example" once we have one designed. -->
+A persistent drawer is an always-visible drawer that can be opened and closed (expanded or collapsed). In its opened state, it looks and behaves like a permanent drawer. It the closed state, it looks and behaves like a navigation rail. The drawer is typically opened and closed by clicking an icon in the header. It can also be opened momentarily on desktop applications by hovering with the mouse. Use this variation if you have:
 
-<DemoCard
-    repository={'side-navigation'}
-    angular={'https://pxblue-components.github.io/angular/?path=/story/components-drawer--with-full-config'}
-    react
-/>
+-   fewer than seven top-level pages
+-   primary pages with labels that require complex icons or may be too long to wrap effectivly
```

Comment:
> primary pages with labels that require complex icons or may be too long to wrap effectivly

I don't quite get what you mean by "complex icons" and "too long to wrap effectively"

Reply:
complex icons meaning icons that are very difficult to understand the meaning at first look. for example we have a bunch of products that use very industry specific terminology and using stock icons doesnt work. in these cases we have to create new icons or use icons in places where they normally wouldnt make sense

Reply:
I see. What is "too long to wrap effectively" here?